server.js

```diff
@@ -1,17 +1,17 @@
 const express = require("express");
-const path = require("path");
-const PORT = process.env.PORT || 3001;
+const PORT = process.env.PORT || 6003;
```

Comment:
I think this port needs to stay as `3001` as thats the hard-coded value in the client's `package.json`